tests/assign-array-to-string/assign_array_to_string.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Minor nit. These are test data and not intended to be ran ... but the shebang is inconsistent.